readthedocs/projects/tasks.py

```diff
@@ -335,7 +335,7 @@ def setup_vcs(self):
         self.setup_env.update_build(state=BUILD_STATE_CLONING)
 
         self._log(msg='Updating docs from VCS')
-        update_imported_docs(self.version.pk)
+        update_imported_docs(self.version.pk, self.setup_env)
```

Comment:
Would like to see kwargs here. I'd love to get to a world where we're explicitly using kwargs in all function/method calls, at least until we can add type checking to the calls :)

Reply:
`update_imported_docs` is a function outside this `UpdateDocsTask` class (it's used from other places). See https://github.com/rtfd/readthedocs.org/pull/3520#discussion_r161767055